packages/beacon-node/src/metrics/metrics/beacon.ts

```diff
@@ -355,5 +355,21 @@ export function createBeaconMetrics(register: RegistryMetricCreator) {
       name: "beacon_weak_head_detected_total",
       help: "Detected current head block is weak. May reorg it out when proposing next slot. See proposer boost reorg for more",
     }),
+
+    // Validator activity metrics
+    validatorActivity: {
+      pendingDeposits: register.gauge({
+        name: "beacon_pending_deposits",
+        help: "Total count of pending deposits",
```

Comment:
using wording "total count" here is kinda misleading since it's not a incrementing counter